<!-- GETTING STARTED -->
A transcriptome fingerprint analysis.

# Introduction
Fig 1. Schematic of the workflow for transcriptome analysis of T1D patients. Whole blood from T1D patients (n=39) and healthy donors (n=43) were collected on two hospitals following standardized protocol. Differential expression (DE) analysis performed with classical methods (i.e. DESeq2, wilcox, roc, t) was applied to this dataset to derive putative T1D gene signatures. Further, a previously developed method based on independent component analysis (ICA) was employed to generate enriched signatures. For gene signatures assessment, the initial sample set was bootstrapped 100 times by selecting 50% of samples. 
![](https://blogger.googleusercontent.com/img/b/R29vZ2xl/AVvXsEgZ3R1rw2OwiLzjVlqCIPf9GT4Nt_ud4ewOzcesCcWYLwJ6L5wBeaQ0CqY8Wt4V8Q_bANHpwg9X6BpmYVsX-ciVX_WQPKGQ1avLf6Mtw_TcopcjXlnZn8yX2vk26BC_dY5rX64LG4VpytwGT1N1NALOBnZGdjTVlG1ruDQMMLpUl3OY3P3wRGMLcgtgwnA/s712/Figures.png)

# Differential analysis
Fig 2. Description of a putative gene signature for T1D. Different expression (DE) analysis were performed by 4 methods (i.e. roc, t, wilcox, DESeq2), and the union of the resultant gene sets was computed. (A) Overlap of the DE gene sets and the p-value threshold used in each corresponding method are shown. (B) Pathway enrichment analysis of the union of DE genes in which the k=4 gene clusters were evaluated independently (see Materials and Methods for further information). (C) Heatmap presentation of the union of DE genes for all samples in which the k=4 gene clusters are delineated by colors.
![](https://blogger.googleusercontent.com/img/b/R29vZ2xl/AVvXsEgTL0CXPGaceClDQ_XzeWSKWc1JKEki7VXs65XGV5xkTSYovOwdLvHA-JydrTexvk-9WerUaP5dVN2H5-_rtbJdnnbJQsdtukDRFjzx5ApLSdhosiWc5pBXv_hTS6BgvMVQaEOv548-hLp87hSeV7ExUe6U4HBBMGQc3hRL_iFIRaRQhB4dmV3gncvYgXY/s906/Figures2.png)

# Figreprint analysis of trascriptome
Fig 2. Description of a putative gene signature for T1D. Different expression (DE) analysis were performed by 4 methods (i.e. roc, t, wilcox, DESeq2), and the union of the resultant gene sets was computed. (A) Overlap of the DE gene sets and the p-value threshold used in each corresponding method are shown. (B) Pathway enrichment analysis of the union of DE genes in which the k=4 gene clusters were evaluated independently (see Materials and Methods for further information). (C) Heatmap presentation of the union of DE genes for all samples in which the k=4 gene clusters are delineated by colors.
![](https://blogger.googleusercontent.com/img/b/R29vZ2xl/AVvXsEiwSxnoyu5X-Q6uOoi1P-5fwz52Lqdu-6VBUt-GQQcXU1rvXdc79-eFGqbsM7V2Dn_OeE4FL2F6ez4xjgCL-rP20CbPdkbw6eA3wvjpVJsjXH8X8nwvKfofssru_ZJkc6SjkyuWfjx7_8dgVFHDlyMS8gbGVbAVEp_LprBaWR1QGETS-O9ZDZb5dyQRmNQ/s688/Figures4.png)
