<!-- GETTING STARTED -->
A transcriptome fingerprint analysis.

# Introduction
Workflow for transcriptome analysis of T1D patients. Whole blood from T1D patients (n=39) and healthy donors (n=43) were collected on two hospitals following standardized protocol. Differential expression (DE) analysis performed with classical methods (i.e. DESeq2, wilcox, roc, t) was applied to this dataset to derive putative T1D gene signatures. Further, a previously developed method based on independent component analysis (ICA) was employed to generate enriched signatures. For gene signatures assessment, the initial sample set was bootstrapped 100 times by selecting 50% of samples. 

# Differential analysis
Fig 2. Description of a putative gene signature for T1D. Different expression (DE) analysis were performed by 4 methods (i.e. roc, t, wilcox, DESeq2), and the union of the resultant gene sets was computed. (A) Overlap of the DE gene sets and the p-value threshold used in each corresponding method are shown. (B) Pathway enrichment analysis of the union of DE genes in which the k=4 gene clusters were evaluated independently (see Materials and Methods for further information). (C) Heatmap presentation of the union of DE genes for all samples in which the k=4 gene clusters are delineated by colors.
![](https://blogger.googleusercontent.com/img/b/R29vZ2xl/AVvXsEgTL0CXPGaceClDQ_XzeWSKWc1JKEki7VXs65XGV5xkTSYovOwdLvHA-JydrTexvk-9WerUaP5dVN2H5-_rtbJdnnbJQsdtukDRFjzx5ApLSdhosiWc5pBXv_hTS6BgvMVQaEOv548-hLp87hSeV7ExUe6U4HBBMGQc3hRL_iFIRaRQhB4dmV3gncvYgXY/s906/Figures2.png)

# Figreprint analysis of trascriptome
Fig 4. Analysis of the gene cluster fingerprint from T1D samples against healthy samples. The k=4 clusters from each of the enriched signatures were detected following hierarchical clustering. Based on KEEG enrichment analysis, a KEEG label was attributed to each of the k=4 clusters in each of the signatures. A collapsing procedure was created to merge clusters when they present equal KEEG label and have same direction of regulation (i.e. up or down regulation) . After collapsing the clusters, the expressions of all genes within same cluster are averaged. Moreover, a  criteria is applied to select clusters for further evaluation. The clusters selection criteria is based on 1) linearity test between expression and diagnostic and 2) averaged expression difference between groups (T1D against healthy) (see Materials and Methods for further information). The expression for the selected clusters over all samples was also used to detect samples clusters. (A) Gene cluster fingerprint for most prominent T1D samples against most prominent healthy samples. (B) PCA of selected clusters over all samples showing k=3 clusters and their corresponding purity. Healthy samples from the cluster represented in green (96% healthy samples) are considered the most prominent healthy, whilst T1D samples from cluster represented in grey (92% T1D samples) are considered the most prominent T1D. Samples in the in the cluster represented by darkgreen color (42% T1D, 58% healthy samples) are considered non-prominent. (C) Gene cluster fingerprints from non prominent samples.
![](https://blogger.googleusercontent.com/img/b/R29vZ2xl/AVvXsEiwSxnoyu5X-Q6uOoi1P-5fwz52Lqdu-6VBUt-GQQcXU1rvXdc79-eFGqbsM7V2Dn_OeE4FL2F6ez4xjgCL-rP20CbPdkbw6eA3wvjpVJsjXH8X8nwvKfofssru_ZJkc6SjkyuWfjx7_8dgVFHDlyMS8gbGVbAVEp_LprBaWR1QGETS-O9ZDZb5dyQRmNQ/s688/Figures4.png)

# Bayesian networks from figreprint analysis of trascriptome
Fig 5. Bayesian interactions among the cluster expression and clinical parameters. (A) The expression for the selected clusters over all samples are ordered according to cluster dendogram. (B) Black-and-white code of clinical parameters are shown to visually investigate possible obvious correlation with gene clusters expression. (C) Bayesian network structure learning was applied to unveil interactions between and among cluster expression and clinical parameters. Algorithm Interleaved Incremental Association (Inter-IAMB) was used for Bayesian network structure learning. The edges detected by Inter-IAMB were further tested for significance of association between variables. The tests and the p-values were adjusted according to the type of the variable (numerical or categorical) (see Materials and Methods for further information). In blue, all the associations that were detected by both Inter-IAMB and classical statistics (p-value < threshold). In
red, the association that was detected by Inter-IAMB but is not significant according to classical statistics.
![](https://blogger.googleusercontent.com/img/b/R29vZ2xl/AVvXsEi9xysSwU3MvBpwE2-wanC5HadwNvX9QZJhoqb7ldlgNOdWB4KTYcu8W4LD5jamOaMa9Sbnb1ZgnxElh473ioCFheRngxJa9XOYalRV7qhxMdqF0DWKq4ajQI8GDPUWony5Y5QkCxBMgqw-_fx85nv8_WUWkskJFKBAXnJ5FVOQgkdEI4f-qmWgWEyT7cI/s758/Figures5.png)

